core/src/epicli/data/common/defaults/configuration/elasticsearch-curator.yml

```diff
@@ -2,7 +2,6 @@ kind: configuration/elasticsearch-curator
 title: Elasticsearch Curator
 name: default
 specification:
-  curator_version: "5.8.3"
```

Comment:
In my opinion it could be nice to move the version from configuration section to the defaults inside of the curator role.

Reply:
And this is done here 😉